external/vcpkg_custom_data/triplets/x64-windows-static-md.cmake

```diff
@@ -0,0 +1,3 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
```

Comment:
Copyrights on all of these files, too, please.

Reply:
Added to all the cmake triplet files. 
Vcpkg response files don't accept comments in text files as far as I know. Couldn't add copyright to those files.